tutor/resources/styles/variables/colors.less

```diff
@@ -41,44 +41,53 @@
 //  * These must be kept in sync with the books in src/flux/book-appearance-codes.coffee
 @openstax-book-appearance-codes: default physics hs_physics college_physics biology ap_biology concepts_biology college_biology principles_economics macro_economics micro_economics intro_sociology anatomy_physiology;
 
```

Comment:
We could probably move these into `variables/books-content.less` or something.  Not really sure where, but feels like they're pretty much taking over the `colors` file.